pkg/server/chaosd/network.go

```diff
@@ -357,7 +360,8 @@ func checkPortIsListened(port string) (bool, error) {
 
 func (s *Server) recoverPortOccupied(attack *core.NetworkCommand, uid string) error {
 
-	c := fmt.Sprintf("lsof -i:%s | awk '{print $2}' | grep -v PID | xargs kill -9", attack.Port)
+	//c := fmt.Sprintf("lsof -i:%s | awk '{print $2}' | grep -v PID | xargs kill -9", attack.Port)
+	c := fmt.Sprintf("kill -9 %d", attack.PortPid)
```

Comment:
https://github.com/chaos-mesh/chaosd/blob/a8213ea516a4ee33d12bd16b92f71eb292a71932/pkg/server/chaosd/stress.go#L102

it is better to add a judgement before kill it, make sure it is the process of httpStartTool